### PR TITLE
Changed login access for local users

### PR DESCRIPTION
**Pull request by sullia**

### CODE REVIEW

File: ARP.Authentication/LdapHelper.cs [hunk 1]
```diff
@@ -30,42 +30,73 @@ namespace garb.Authentication
...
-				}
-			}
-			finally
-			{
-			}
-
-			return false;
-		}
-	}
+            if (username.Equals(LocalAdminName) && password.Equals(LocalAdminPwd) ||
```
Comment (apoblock) [inline ARP.Authentication/LdapHelper.cs]: Please check first if those values are not null or empty - when they are not set they should not allow anyone to login with empty creds.

File: ARP.Authentication/LdapHelper.cs [hunk 2]
```diff
@@ -30,42 +30,73 @@ namespace garb.Authentication
...
+                username.Equals(LocalReadOnlyName) && password.Equals(LocalReadOnlyPwd))
+            {
+                var userRepo = _unitOfWork.UserRepository;
```
Comment (apoblock) [inline ARP.Authentication/LdapHelper.cs]: This section is duplicated with the one inside the Ldap block - please try to consolidate.

File: ARP.Authentication/LdapHelper.cs (hunk 1 above)
Comment (sullia) [reply ARP.Authentication/LdapHelper.cs]: Changed LdapHelper.cs to check for null or empty username and password.

File: ARP.Authentication/LdapHelper.cs (hunk 2 above)
Comment (sullia) [reply ARP.Authentication/LdapHelper.cs]: Created CreateRepro method to avoid duplication.

File: ARP.Authentication/LdapHelper.cs [hunk 3]
```diff
@@ -30,42 +30,79 @@ namespace garb.Authentication
...
-			finally
-			{
-			}
-
-			return false;
-		}
-	}
+            try
+            {
+                if (!string.IsNullOrEmpty(username) || !string.IsNullOrEmpty(password))
```
Comment (apoblock) [inline ARP.Authentication/LdapHelper.cs]: shouldn't it be '&&' ?

Comment (sullia) [reply ARP.Authentication/LdapHelper.cs]: Yep. It should be &&. Changed. Thanks.

**Pull request merged**